Assets/Locales/SongEditMenu_es.asset

```diff
@@ -368,5 +368,13 @@ MonoBehaviour:
       it instead?
     m_Metadata:
       m_Items: []
+  - m_Id: 78
+    m_Localized: "Archivos exportados con \xE9xito a Quest"
```

Comment:
As helpful as this is the way we have crowdin set up this won't get uploaded and will have to be entered there manually anyway